src/officehours_api/notifications.py

```diff
@@ -14,7 +14,7 @@
 
 
 def notify_next_in_line(next_in_line: Meeting):
-    if not (settings.TWILIO_ACCOUNT_SID and settings.TWILIO_AUTH_TOKEN and settings.TWILIO_PHONE_FROM):
+    if not (settings.TWILIO_ACCOUNT_SID and settings.TWILIO_AUTH_TOKEN and settings.TWILIO_MESSAGING_SERVICE_SID):
```

Comment:
I think I mentioned this in my review of the initial notifications PR, but I'll say it again in case I failed to communicate well before. While the computational cost of this is probably low, it would make more sense to me to have these variable checks determine (somewhere else) whether to look for the relevant signals in the first place, rather than have the signals still function and then bail out of these functions when they trigger. That would also reduce some duplication. Maybe you could just move this check to where you imported the `notifications` module in `models.ply`.

Reply:
Since this is only really relevant for local development or testing environments where Twilio isn't configured, it's also not a big deal, probably in the nit-pick category.

Reply:
That's a good idea. That module will now assume those settings are configured. I also managed to bring `twilio` up to the module level. I could go even farther in reducing duplication, but I think it would add more complexity than it removes. Things still function & tests still pass with & without the settings configured.

Reply:
Looks good to me!